pages/style-guide/prose.md

```diff
@@ -0,0 +1,111 @@
+<script>{
+    "title": "Prose Style Guide"
+}</script>
+
+## Formatting Conventions
+
+Articles about all content (sites, README, docs, etc) are authored with [GitHub Flavored Markdown](http://github.github.com/github-flavored-markdown/).
+
+### Article Header Metadata
+
+Each article should have the following header (see below as some metatags are optional):
+
+```
```

Comment:
Enable the right syntax highlighting for this code block. (https://github.com/adam-p/markdown-here/wiki/Markdown-Cheatsheet#code-and-syntax-highlighting)